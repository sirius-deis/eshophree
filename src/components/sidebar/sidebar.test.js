import { render, screen } from "@testing-library/react";
import Sidebar from "./sidebar";

const title = "Some title"

const elements = [
  { title: "Home", href: "/" },
  { title: "About", href: "/about" },
  { title: "Contact", href: "/contact" },
]


describe("Sidebar component", () => {
  it("should match snapshot", () => {
    const { container } = render(<Sidebar elements={elements} heading={title}/>);
    expect(container).toMatchSnapshot();
  });
  it("should render heading", () => {
    render(<Sidebar elements={elements} heading={title} />);
    expect(screen.getByText(title)).toBeInTheDocument();
  });
});