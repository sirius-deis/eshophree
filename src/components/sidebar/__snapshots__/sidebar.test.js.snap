// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB dunMaC"
  >
    <h3>
      Some title
    </h3>
  </div>
</div>
`;
